### PR TITLE
Sergio/pokerhands

### PR DESCRIPTION
**Pull request by SerJimmy**

### CODE REVIEW

File: src/test/java/org/fundacionjala/at15/katas/pokerhands/sergio/ThreesomeIdentifierTest.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ThreesomeIdentifierTest {
```
Comment (J0c4) [inline src/test/java/org/fundacionjala/at15/katas/pokerhands/sergio/ThreesomeIdentifierTest.java:11]: Nice work on the tests

**Review by J0c4 — CHANGES_REQUESTED** [review]
Nice job, please check my comments

File: .gitignore [hunk 2]
```diff
@@ -25,4 +25,5 @@ hs_err_pid*
 # gradle
 .gradle
 bin/
-build/
\ No newline at end of file
+build/
+.idea
```
[the remark this answers is not in the document]
Comment (SerJimmy) [reply .gitignore:29]: done

File: src/main/java/org/fundacionjala/at15/katas/pokerhands/sergio/Croupier.java [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+    public String getFirstDecision() {
+        return firstDecision;
+    }
+
+    public void doFirstDecision() {
+        this.player1Points = new PointsAssigner(player1.getPokerHand());
+        this.player2Points = new PointsAssigner(player2.getPokerHand());
+
+
```
[the remark this answers is not in the document]
Comment (SerJimmy) [reply src/main/java/org/fundacionjala/at15/katas/pokerhands/sergio/Croupier.java:35]: they are not empty, I need those ones to make the first comparation

File: src/main/java/org/fundacionjala/at15/katas/pokerhands/sergio/Croupier.java [hunk 4]
```diff
@@ -0,0 +1,35 @@
+package org.fundacionjala.at15.katas.pokerhands.sergio;
+
+public class Croupier {
```
[the remark this answers is not in the document]
Comment (SerJimmy) [reply src/main/java/org/fundacionjala/at15/katas/pokerhands/sergio/Croupier.java:3]: thanks, decide the winner, but Croupier is in French, I don't know if works

File: src/main/java/org/fundacionjala/at15/katas/pokerhands/sergio/Identifier.java [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+    protected boolean isFlush = false;
+    protected boolean isStraightFlush = false;
+    protected boolean isPoker = false;
+    protected boolean isFullHouse = false;
+    protected boolean isThreesome = false;
+    protected boolean isDoublePair = false;
+    protected boolean isOnlyPair = false;
+    protected boolean isSimpleHand = false;
+
+    public abstract void theIdentification(List<Card> newPokerHand);
```
[the remark this answers is not in the document]
Comment (SerJimmy) [reply src/main/java/org/fundacionjala/at15/katas/pokerhands/sergio/Identifier.java:19]: I tried do a refactor, but a lot of code brake if I try it, I'll asume this bug.

File: src/main/java/org/fundacionjala/at15/katas/pokerhands/sergio/Main.java [hunk 6]
```diff
@@ -0,0 +1,23 @@
+package org.fundacionjala.at15.katas.pokerhands.sergio;
+
+import java.util.*;
+
+public class Main {
```
[the remark this answers is not in the document]
Comment (SerJimmy) [reply src/main/java/org/fundacionjala/at15/katas/pokerhands/sergio/Main.java]: done

File: src/main/java/org/fundacionjala/at15/katas/pokerhands/sergio/Identifier.java [hunk 7]
```diff
@@ -0,0 +1,92 @@
+package org.fundacionjala.at15.katas.pokerhands.sergio;
+
+import java.util.List;
+
+public abstract class Identifier {
```
[the remark this answers is not in the document]
Comment (SerJimmy) [reply src/main/java/org/fundacionjala/at15/katas/pokerhands/sergio/Identifier.java:5]: thanks, though I couldn't find a way to use interfaces.

File: src/test/java/org/fundacionjala/at15/katas/pokerhands/sergio/ThreesomeIdentifierTest.java (hunk 1 above)
Comment (SerJimmy) [reply src/test/java/org/fundacionjala/at15/katas/pokerhands/sergio/ThreesomeIdentifierTest.java:11]: thanks, 99% code coverage

**Review by J0c4 — APPROVED** [review]
Great job!

About the empty lines, I actually meant when there are lines without statements that not necessarily helps to code readability, for example, if you have
```
int a = 10
int b = 30


int c = 36
```
You have unneeded empty lines between declaration of variable "b" and "c". If you remove them, it looks like this
```
int a = 10
int b = 30
int c = 36
```

That way, you get a cleaner code.

Your class and method names are very good. Few exceptions where you start method names with "the" is where maybe can be changed to a verb that describes better what is the method doing (example: "calculate", "estimate", etc)

**Review by J0c4 — APPROVED** [review]

**Pull request merged**